tests/cli/preprocess_script_tests.py

```diff
@@ -0,0 +1,171 @@
+import cv2
+import nose.tools
+import numpy as np
+import os
+from testfixtures import TempDirectory
+import unittest
+
+import micro_dl.cli.preprocess_script as pp
+import micro_dl.utils.aux_utils as aux_utils
+
+
+class TestPreprocessScript(unittest.TestCase):
+
+    def setUp(self):
+        """
+        Set up a directory with some images to resample
+        """
+        self.tempdir = TempDirectory()
+        self.temp_path = self.tempdir.path
+        self.output_dir = os.path.join(self.temp_path, 'out_dir')
+        self.tempdir.makedir(self.output_dir)
+        # Start frames meta file
+        self.meta_name = 'frames_meta.csv'
+        self.frames_meta = aux_utils.make_dataframe()
+        # Write images
+        self.time_idx = 5
+        self.pos_idx = 7
+        self.im = 1500 * np.ones((30, 20), dtype=np.uint16)
+
+        for c in range(4):
+            for p in range(self.pos_idx, self.pos_idx + 3):
+                for z in range(5):
+                    im_name = aux_utils.get_im_name(
+                        channel_idx=c,
+                        slice_idx=z,
+                        time_idx=self.time_idx,
+                        pos_idx=p,
+                        ext='.png',
+                    )
+                    cv2.imwrite(os.path.join(self.temp_path, im_name),
+                                self.im + c * 100)
+                    self.frames_meta = self.frames_meta.append(
+                        aux_utils.parse_idx_from_name(im_name),
+                        ignore_index=True,
+                    )
+        # Write metadata
+        self.frames_meta.to_csv(
+            os.path.join(self.temp_path, self.meta_name),
+            sep=',',
+        )
+        self.pp_config = {
+            'output_dir': self.output_dir,
+            'input_dir': self.temp_path,
+            'channel_ids': [0, 1, 3],
+            'resample_scale': 0.2,
```

Comment:
this option is not used with the last change, we moved this param to resize['scale_factor']!